plugins/destination/s3/client/spec.go

```diff
@@ -12,13 +12,19 @@ const (
 )
 
 type Spec struct {
-	Bucket   string     `json:"bucket,omitempty"`
-	Path     string     `json:"path,omitempty"`
-	Format   FormatType `json:"format,omitempty"`
-	NoRotate bool       `json:"no_rotate,omitempty"`
+	Bucket         string     `json:"bucket,omitempty"`
+	Path           string     `json:"path,omitempty"`
+	Format         FormatType `json:"format,omitempty"`
+	IncludeHeaders bool       `json:"include_headers,omitempty"`
```

Comment:
This setting only applies to the CSV format: maybe we should introduce a `format_spec` or something along those lines to capture more settings in the future?